wordpress-mobile/WordPress-Android
# Site Monitor: Implement web view basics

Part of #20035, and #20036, and #20037

This PR implements the basics for the SiteMonitor feature area. 

Review the PR: 
- The first few commits include housekeeping tasks
- The bulk of this PR are the commits on Jan 28, 2024

Open Items that will be addressed in a separate PR
 (1) No unit tests 
(2) Color are off in the tabs :facepalm: 
(3) Tracking when selecting tabs 
(4**) Add a PTR if needed (alternative is leave tab and come back)
(5**) If possible, retaining the state of the webview contents. This may be too complex to accomplish by our Tuesday deadline. Feel free to tackle any of these after the PR has been merged. If not, I will tackle them on Monday when I get in


Metrics | PHP Logs | Web Server Logs
-- | -- | --
<img width="250" height="500" alt="Alt desc" src="https://github.com/wordpress-mobile/WordPress-Android/assets/506707/5618e456-30bc-4034-b548-9fc2bba78904"> | <img width="250" height="500" alt="Alt desc" src="https://github.com/wordpress-mobile/WordPress-Android/assets/506707/fd67a2ac-7acc-4792-8d58-d3930ec03518"> | <img width="250" height="500" alt="Alt desc" src="https://github.com/wordpress-mobile/WordPress-Android/assets/506707/bbf48eb4-5174-4af8-be2b-07a113819cbd">


-----

## To Test:
- Install and launch the JP app from this PR
- Login with an account that has an atomic with admin access
- Select ^^ that site
- Navigate to My Site > More > 
- ✅ Verify that the Site Monitoring Item is shown
- Tap on the Site Monitoring item
- ✅ Verify that the Site Monitoring view is shown
- ✅ Verify you can navigate from tab to tab

-----


## Regression Notes

1. Potential unintended areas of impact
    - Site Monitor tabs are not shown

2. What I did to test those areas of impact (or what existing automated tests I relied on)
    - Manual testing

3. What automated tests I added (or what prevented me from doing so N/A

-----

## PR Submission Checklist:

-----

## UI Changes Testing Checklist:
